distmap

```diff
@@ -235,31 +241,49 @@ GetOptions(
 	"disable-zipped-output" 		=>\$nozip,
 	"hadoop-home=s" 			=>\$hadoop_home,
 	"mappers-exe-archive=s" 		=>\$mappers_exe_archive,
-	
-	
-	"mapper=s" 				=> \@mapper,		# to check	
+
+
+	"mapper=s" 				=> \@mapper,		# to check
 	"mapper-path=s"				=> \@mapper_path,
 	"picard-jar=s"				=> \$picard_jar,
 	"picard-mergesamfiles-jar=s"		=> \$picard_mergesamfiles_jar,	# to ckeck
 	"picard-sortsam-jar=s"			=> \$picard_sortsam_jar,	# to check
+	"readtools=s"         => \$readtools,
 	"mapper-args=s"				=> \@mapper_args,
 	"bwa-sampe-args=s"			=> \$bwa_sampe_args,
-	"output-format=s"			=> \$output_format,					
+	"output-format=s"			=> \$output_format,
```

Comment:
We should open an issue to support at some point the CRAM output format.

Reply:
yes

Reply:
Done.